plugins/plugin_repeat.py

```diff
@@ -47,10 +38,10 @@ def check_for_flag_words(self, message):
         cnt = Counter()
         delims = '!"#$%&()*+,./:;<=>?@[\\]^_`{|}~\t\n\x0b\x0c\r '
         pattern = r"[{}]".format(delims)
-        message.lower()
-        message_array = re.split(pattern, message)
+        message_array = re.split(pattern, message.lower())
         for word in message_array:
-            if word in self.words:
-                cnt[word] += 1
-
+            word_v2 = word.replace(u"\u2019", "")
+            word_v3 = word_v2.replace("'", "")
```

Comment:
a nit: you can chain these two `replace` calls so we only have to save the word once:
`formatted_word = word.replace(u"\u2019", "").replace("'", "")`